controllers/snapshot/snapshot_adapter.go

```diff
@@ -356,6 +375,23 @@ func (a *Adapter) createIntegrationPipelineRun(application *hasv1alpha1.Applicat
 	if err != nil {
 		return err
 	}
+	//Copy PipelineRun PAC annotations/labels from applicaitonSnapshot to integration test PipelineRuns
```

Comment:
Similar to the above, copying of labels and annotations should happen before the `a.Client.Create(a.context, newRelease)` call on line 374 in order for them to be present on the created object.

Reply:
Updated